app/src/main/res/values/strings.xml

```diff
@@ -313,31 +313,51 @@
     <string name="calculate_calories">Calculate Calories</string>
     <string name="txt_calories_result">%1$s kcal in %2$s %3$s</string>
     <string name="nutrition_carbohydrate">Carbohydrate</string>
+    <string name="calculate_carbohydrate">Calculate Carbohydrate</string>
+    <string name="txt_carbohydrate_result">%1$s %2$s in %3$s %4$s</string>
```

Comment:
For this, you should have a general template. This means that you should only have one version of this string rather than the 20 or so that are there. Since they are all the same string.